eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         artifact: repository
 
   - job: 'Analyze'
-    condition: ne(variables['Skip.Analyze'], 'true')
+    condition: and(succeeded(), ne(variables['Skip.Analyze'], 'true'))
```

Comment:
I believe all the jobs are run in parallel, so would there be any difference between `succeeded()` and `succeededAndFailed()` here?  If not then `succeeded()` is fine with me.

Reply:
I tried to apply a general rule as I was going through which was if a custom condition was defined make sure that a job state condition was also applied. I do actually plan on adding a prep job to all build stages in all pipelines which everything will depend on, so I kinda had that in the back of my head.